website/src/components/Mainhero.astro

```diff
@@ -0,0 +1,31 @@
+
+<html  lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
```

Comment:
@Jawad-sawari this head thing should be removed, we don't need them, it's not logical.

Reply:
@Morteza-Mazrae I don't think in astro components we bring <html> and <head>

only the elements we need, in case of this file, the <div> only we need.

Reply:
@Aslemammad  I removed the whole extra head tags